src/model.d.ts

```diff
@@ -2243,7 +2243,7 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
     this: ModelStatic<M>,
     field: keyof Attributes<M>,
     options: IncrementDecrementOptionsWithBy<Attributes<M>>
-  ): Promise<M>;
+  ): Promise<[M[], number?]>;
```

Comment:
```suggestion
  ): Promise<[affectedRows: M[], affectedCount?: number]>;
```

last request but could you label these tuples ? :)